examples/spdlog/foo.cpp

```diff
@@ -1,6 +1,8 @@
 #include <spdlog/spdlog.h>
+#include <spdlog/sinks/stdout_sinks.h>
```

Comment:
> I have checked that this pull request contains only
.travis.yml/appveyor.yml changes. All other changes send
to https://github.com/ruslo/hunter. Yes

Please send this to https://github.com/ruslo/hunter :
* https://github.com/ruslo/hunter/blob/v0.23.34/examples/spdlog/foo.cpp#L1-L6

I guess it should be part of https://github.com/ruslo/hunter/pull/1588